src/stratis_cli/_parser/_pool.py

```diff
@@ -139,10 +139,50 @@
             func=TopActions.add_cache_devices,
         ),
     ),
+    (
+        "bind-clevis",
+        dict(
+            help="Unlock all of the currently locked encrypted pools",
+            args=[
+                ("pool_name", dict(action="store", help="Pool name")),
+                (
+                    "key_desc",
+                    dict(action="store", help="Key description of existing pool key"),
+                ),
+                ("tang_url", dict(action="store", help="URL of Tang server")),
+                (
+                    "tang_thp",
+                    dict(
+                        action="store",
+                        help="Thumbprint of Tang server verification key",
+                    ),
+                ),
+            ],
+            func=TopActions.bind_clevis,
+        ),
+    ),
+    (
+        "unbind-clevis",
+        dict(
+            help="Unlock all of the currently locked encrypted pools",
+            args=[("pool_name", dict(action="store", help="Pool name"))],
+            func=TopActions.unbind_clevis,
+        ),
+    ),
     (
         "unlock",
         dict(
             help="Unlock all of the currently locked encrypted pools",
+            args=[
+                (
+                    "unlock_method",
+                    dict(
+                        default="keyring",
```

Comment:
You can add "choices" field here. What should the second choice be? Just "clevis"?

Reply:
If we change it to something else, that'll need to be updated in stratisd too.